docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -200,20 +241,19 @@ const styles = StyleSheet.create({
 
 ## Wait for fonts to load
 
-Since your fonts won't be ready right away, it is generally a good practice to not render anything until the font is ready. Instead, you can continue to display the Splash Screen of your app until all fonts have loaded (or an error has been returned). It is done by using [`expo-splash-screen`](/versions/latest/sdk/splash-screen/) package. See the [minimal example](#minimal-example) section on how to use it.
+If you [embed your fonts in your native project](#1-embed-the-font-in-your-native-project), they will be available immediately without any additional code outside of the config plugin. However, if you load your fonts at runtime, they will not be available immediately — so, it is generally a good practice to not render anything until the font is ready. Instead, you can continue to display the Splash Screen of your app until all fonts have loaded (or an error has been returned). It is done by using [`expo-splash-screen`](/versions/latest/sdk/splash-screen/) package. See the [minimal example](#minimal-example) section on how to use it.
 
 ### Load fonts on the web
 
-Sometimes, particularly on the web -- people choose to render their content in a platform default font while their custom font is loading. Alternatively, to render the rest of their content, that doesn't depend on the custom font while the font is loading. These approaches are called FOUT and FOIT and you can read a lot more about them on the web.
+Sometimes, particularly on the web people choose to render their content in a platform default font while their custom font is loading. Alternatively, to render the rest of their content, that doesn't depend on the custom font while the font is loading. These approaches are called FOUT and FOIT and you can read a lot more about them on the web.
 
-In general, these strategies are not recommended for native apps. If you include your fonts in your project, the
-fonts will always be delivered to the user by the time your code is running. The one exception to this is that you may prefer to do this on the web.
+In general, these strategies are not recommended for native apps. If you include your fonts in your project, the fonts will always be delivered to the user by the time your code is running. The one exception to this is that you may prefer to do this on the web.
 
 ## Additional information
 
 You probably don't need to know anything beyond this point to use custom fonts effectively in your app. If you are curious or your use case has not been addressed by the above information, please continue reading.
 
-### Load a remote font directly from the web
+### Loading a remote font directly from the web
```

Comment:
In the whole document, we're using short-form verbs in the headers for consistency instead of gerunds, so I'm not sure why it is introduced for this and the last sub-section.

Reply:
the problem with saying "Load a remote font directly from the web" is that it sounds like we're recommending that people do this, rather than explaining how you can do this

Reply:
I understand it now. Thanks for providing the context 🙏